NthKey/View/Addresses/AddressesView.swift

```diff
@@ -31,23 +34,45 @@ struct AddressesView: View {
                 .padding(.top)
 
                 if model.items.count > 0 {
-                    List {
-                        ForEach(model.items) { item in
-                            AddressView(item: item)
-                                .contentShape(Rectangle())
-                                .onTapGesture {
-                                    model.toggleUsed(for: item)
-                                }
+                    ScrollViewReader { proxy in
+                        List {
+                            ForEach(model.items, id: \.self.id) { item in
+                                AddressView(item: item)
+                                    .contentShape(Rectangle())
+                                    .onTapGesture {
+                                        model.toggleUsed(for: item)
+                                    }
+                                    .id(item.id)
+                                    .listRowBackground(
+                                        VStack {
+                                            if model.addressToScroll == item.address {
+                                                RoundedRectangle(cornerRadius: 20)
+                                                    .stroke(selectionColor, lineWidth: 2)
+                                                    .padding(2)
+                                            }
+                                        }
+                                    )
+                            }
                         }
+                        .onChange(of: model.addressToScroll, perform: { value in
+                            withAnimation {
+                                proxy.scrollTo(value, anchor: .top)
+                            }
+                        })
                     }
                 } else {
                     Text("Go to Settings to add cosigners")
                 }
             }
             .navigationBarTitle("Address")
-            .sheet(isPresented: $showScanner) {
+            .sheet(isPresented: $model.showScanner) {
                 CodeScannerView(codeTypes: [.qr], completion: model.handleScan)
             }
+            .alert(item: $model.scanQRError) { error in
+                Alert(title: Text("Import PSBT error"),
```

Comment:
This title is wrong when scanning an address.

Reply:
fixed in 888f99fb65af1e5fed4c2e51e7aa8e4f20e72181